akka-remote-tests/src/multi-jvm/scala/akka/remote/RemoteFeaturesSpec.scala

```diff
@@ -120,6 +121,28 @@ abstract class RemotingFeaturesSafeSpec
         expectMsgType[ActorIdentity].ref.get.path.address.hasGlobalScope shouldBe false
       }
     }
+
+    "not receive Terminated on stop with watch attempt" in {
+      runOn(second) {
+        system.actorOf(Props(classOf[ProbeActor], probe.ref), "terminating")
+      }
+      runOn(first) {
+        val watcher = system.actorOf(Props(classOf[ProbeActor], probe.ref), "watch-terminating")
+        val terminating = identify(second, "terminating")
+        watcher ! WatchIt(terminating)
+      }
+      enterBarrier("watch-t-attempted")
+
+      runOn(second) {
+        val terminating = identify(second, "terminating")
+        system.stop(terminating)
+      }
+      enterBarrier("t-stopped")
+
+      runOn(first) {
+        probe.expectNoMessage(2.seconds)
+      }
+    }
```

Comment:
yes, that's the right scenario

Reply:
Failing tests are due to it working as expected now for those cases. Updating.

Reply:
good